spec/ssh_scan/policy_manager_spec.rb

```diff
@@ -192,4 +192,25 @@
     end
   end
 
+  context "when checking the minimum ssh_version" do
+    yaml_string = "---\nname: Mozilla Intermediate\nkex:\n" +
+                  "- diffie-hellman-group-exchange-sha256\n" +
+                  "encryption:\n- aes256-ctr\n- aes192-ctr\n" +
+                  "- aes128-ctr\nmacs:\n- hmac-sha2-512\n" +
+                  "- hmac-sha2-256\ncompression:\n- none\n" +
+                  "- zlib@openssh.com\n" +
+                  "references:\n- https://wiki.mozilla.org/Security/Guidelines/OpenSSH\n" +
+                  "auth_methods:\n- publickey\n" +
+                  "ssh_version: 2.0\n"
+    result = {
+                :ssh_version => 1.9
+             }
+
+    it "should load all the attributes properly" do
+      policy = SSHScan::Policy.from_string(yaml_string)
+      policy_manager = SSHScan::PolicyManager.new(result, policy)
+      expect(policy_manager.minimum_ssh_version()).to eql(2.0)
```

Comment:
Same vein as the method criticism, I think we need more deliberate methods here.  We're using one method for multiple uses and I don't think it's direct enough.